src/sage/rings/polynomial/multi_polynomial_sequence.py

```diff
@@ -771,16 +791,20 @@ def coefficient_matrix(self, sparse=True):
         #construct dictionary for fast lookups
         v = dict( zip( m , range(len(m)) ) )
 
-        from sage.matrix.constructor import Matrix
-
-        A = Matrix( R.base_ring(), nf, nm, sparse=sparse )
+        A = Matrix( R.base_ring(), nf, nm, sparse=sparse)
 
         for x in range( nf ):
             poly = f[x]
             for y in poly.monomials():
                 A[ x , v[y] ] = poly.monomial_coefficient(y)
 
-        return A, Matrix(R,nm,1,m)
+        if vector_form:
+            return A, vector(m)
+
+        deprecation(37027,'the function PolynomialSequence.coeffient_matrix() will'
+                          ' return a vector of monomilas instead of a matrix as second element in tuple in the future,'
+                          ' please use vector_form argument to get vector form of monomilas.')
```

Comment:
`monomilas` -> `monomials`
remove `in tuple`
I'd remove the third line (`, please ... `) completely, to make it shorter.